fog/ledger/server/src/router_handlers.rs

```diff
@@ -246,7 +247,11 @@ where
     // If there's no new store and we don't have enough responses, decrement
     // remaining_retries and loop
     let mut remaining_retries = query_retries;
+    let _timer = BULK_QUERY_REQUESTS.start_timer();
     while remaining_retries > 0 {
+        if remaining_retries < query_retries {
+            QUERY_GROUP_RETRY.inc();
+        }
```

Comment:
I think it would be cleaner to inc the timer after the `break` `if` block further down the loop